src/server.js

```diff
@@ -66,5 +67,6 @@ app.get('/setup', async (req, res) => {
 
 app.post('/login', AuthController.login);
 app.post('/signup', signupController.signup);
+app.post('/submitform', submitformController.submit);
```

Comment:
Make a constants file and add these routes in it.
Make sure to import the routes in the test files also.
For example in file `__tests__/submitForm.test.js` you have used `/submitForm`.
So make sure all the places it is used is replaced with the constants